main_routers/jukebox_router.py

```
@@ -0,0 +1,1111 @@
+    """
+    config_mgr = get_config_manager()
+    jukebox_config = JukeboxConfig(config_mgr)
+    
+    # 去除前导斜杠，防止路径解析问题
+    file_path = file_path.lstrip('/')
+    
+    # 处理 /static/jukebox/ 前缀（自带资源的特殊路径）
+    if file_path.startswith('static/jukebox/'):
+        file_path = file_path.replace('static/jukebox/', '', 1)
+    
+    # 安全检查：确保路径在 jukebox 目录内
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)